contrib/epee/include/net/levin_protocol_handler_async.h

```diff
@@ -521,21 +510,8 @@ class async_protocol_handler
             }
             else
             {
-              invoke_response_handlers_guard.unlock();
-              //use sync call scenario
-              if(!m_wait_count && !m_close_called)
-              {
-                MERROR(m_connection_context << "no active invoke when response came, wtf?");
-                return false;
-              }else
-              {
-                CRITICAL_REGION_BEGIN(m_local_inv_buff_lock);
-                m_local_inv_buff = std::string((const char*)buff_to_invoke.data(), buff_to_invoke.size());
-                buff_to_invoke = epee::span<const uint8_t>((const uint8_t*)NULL, 0);
-                m_invoke_result_code = m_current_head.m_return_code;
-                CRITICAL_REGION_END();
-                m_invoke_buf_ready = true;
-              }
+              MERROR("Received levin response but have no invoke handlers");
+              return false;
```

Comment:
Does this ever trigger when running monerod on mainnet or testnet?

Reply:
I've never seen this message, this would indicate some error elsewhere that needs correcting.